src/lib/Libwin/passwd.c

```diff
@@ -4956,18 +4950,17 @@ getpwnam(const char *name)
 	struct  passwd *pwdp = NULL;
 	int found = 0;
 
-	if(user_num == 0){
+	if(user_num == 0)
```

Comment:
Same comment as above. We should not need to check for user_num == 0 in each function, rather do a CLEAR_HEAD at server startup. That way you do not even need this variable. There is also a formatting error: Space between if and (